cli/fathom_web/commands/vectorize.py

```diff
@@ -0,0 +1,315 @@
+from functools import partial
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+import os
+import pathlib
+import shutil
+import signal
+import subprocess
+from tempfile import TemporaryDirectory
+from threading import Thread
+import time
+import zipfile
+
+from click import argument, command, option, Path, progressbar
+from selenium import webdriver
+
+
+class GracefulError(RuntimeError):
+    """Raised when encountering error that allows for a graceful shutdown."""
+    pass
```

Comment:
`pass` is just a hint for the lexer. You actually don't need it when you have anything else indented at this level, like your docstring.

Reply:
I didn't know you could do that for a class definition. Very cool.